src/utils/openmc/universe.py

```diff
@@ -911,8 +880,24 @@ def create_xml_subelement(self, xml_element):
         if not test is None:
             return
 
-        lattice_subelement = \
```

Comment:
I assume you don't think there is any reason to abstract a portion of this code from both the ``RectLattice`` and ``HexLattice`` into the ``Lattice`` parent class?